tasks/pre-commit-check.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pre-commit-check
+  labels:
+    app: thoth-ci
+spec:
+  params:
+    - name: pr_number
+      description: Pull Request number.
+    - name: pipelinerun_name
+      description: pipelinerun associated.
+  resources:
+    inputs:
+      - name: s2i-thoth
+        type: image
+      - name: pr
+        type: pullRequest
+      - name: repo
+        type: git
+    outputs:
+      - name: pr
+        type: pullRequest
+
+  steps:
+    - name: git-init-op
+      image: $(resources.inputs.s2i-thoth.url)
+      workingDir: /workspace/repo
+      securityContext:
+        privileged: true
+      script: |
+        cp -r /workspace/pr/ /workspace/output/
+        git fetch origin pull/$(params.pr_number)/head:workbranch
+        git checkout workbranch
+
+    - name: run-pre-commit
+      image: quay.io/thoth-station/thoth-coala:v0.10.2  # TODO: replace with image for pre-commit
```

Comment:
Awesome work.
The build of the pre-commit can be done here: https://github.com/thoth-station/zuul-infra

Reply:
```suggestion
      image: quay.io/thoth-station/thoth-precommit:v0.10.3 
```